yarn.lock

```diff
@@ -2288,6 +2332,33 @@
     semver "^7.3.7"
     tsutils "^3.21.0"
 
+"@typescript-eslint/typescript-estree@5.52.0":
```

Comment:
some new dupes here. try `npx yarn-deduplicate`

Reply:
TIL about `yarn-deduplicate`, but running it made no changes.